changelogs/fragments/1830-valmod_docmissingtype_batch1.yml

```diff
@@ -0,0 +1,9 @@
+bugfixes:
+  - kibana_plugin - ``state`` parameter choices must use ``list()`` in python3 (https://github.com/ansible-collections/community.general/pull/1830).
+  - elasticsearch_plugin - ``state`` parameter choices must use ``list()`` in python3 (https://github.com/ansible-collections/community.general/pull/1830).
+  - riak - parameters ``wait_for_handoffs`` and ``wait_for_ring`` are ``int`` but the default value was ``false`` (https://github.com/ansible-collections/community.general/pull/1830).
+  - logstash_plugin - wrapped ``dict.keys()`` with ``list`` for use in ``choices`` setting (https://github.com/ansible-collections/community.general/pull/1830).
+  - iso_extract - use proper alias deprecation mechanism for ``thirsty`` alias of ``force`` (https://github.com/ansible-collections/community.general/pull/1830).
+minor_changes:
+  - runit - removed unused code (https://github.com/ansible-collections/community.general/pull/1830).
```

Comment:
```suggestion
```

Reply:
This code removal is still there. Doesn't that warrant a changelog ?

Reply:
That part is already in the changelog fragment in the line below :)